indexing-service/src/main/java/org/apache/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -314,9 +314,10 @@ public TaskStatus call()
                         if (context != null) {
                           for (String propName : context.keySet()) {
                             if (propName.startsWith(CHILD_PROPERTY_PREFIX)) {
+                              Object contextValue = task.getContextValue(propName);
                               command.addSystemProperty(
                                   propName.substring(CHILD_PROPERTY_PREFIX.length()),
-                                  task.getContextValue(propName)
+                                  contextValue == null ? null : contextValue.toString()
```

Comment:
if the `contextValue` is null, does it make sense to call the `addSystemProperty` method?

Reply:
Yeah, that's true. We can just skip the `addSystemProperty` if the value is null.